devbin/model_extractor/model_state_parser.py

```diff
@@ -0,0 +1,78 @@
+import json
+import sys
+import sh
+
```

Comment:
I think this file should have:

1. A copyright header.
2. A comment saying what the prerequisites are.  For example, it requires `jq` be installed, which it isn't by default on macOS.  And does it require Python 2 or Python 3 or does it work in both?